package.json

```diff
@@ -14,6 +14,8 @@
     "mongo_queue_processor": "node extensions/mongoProcessor/mongoProcessorTask.js",
     "garbage_collector": "node extensions/gc/service.js",
     "test": "mocha --recursive tests/unit",
+    "ft_test:models": "mocha --recursieve $(find tests/functional/models -name '*.js')",
```

Comment:
I would rather call suffix with `:lib`, as "model" usually applies specifically to object mapping to storage underneath, where we're actually testing some general library code (including models)

Reply:
what do you think about the directory name? I wasn't sure honestly, so named it"models".

Reply:
Yes, lib is good as it matches the suffix it's less confusing.

Reply:
It also matches the directory where the code being tested lies, which can help to link the code and tests together.

Reply:
Great, thank you jonathan!